composer.json

```diff
@@ -24,12 +24,12 @@
         "docs": "https://github.com/leanphp/phpspec-code-coverage#phpspec-code-coverage"
     },
     "require": {
-        "php": "~5.6||~7.0",
-        "phpspec/phpspec": "~3.0",
+        "php": "^7.0",
+        "phpspec/phpspec": "^4.0",
```

Comment:
What about "phpspec/phpspec": "~3.0||~4.0"?

Reply:
There are previous releases which already cater to PhpSpec 3 - maybe you could use those if you can't upgraded to PhpSpec 4?

Reply:
@Landerstraeten the update would be for PHPSpec v4. Allowing both versions would cause problems when people try to install updated version with 3.0 versions (and we can only make sure it's compatible with v4). After this PR is merged, the composer will be able to detect which version of `phpspec-code-coverage` to install depending on which PhpSpec version is used. For PhpSpec v3, the older version of the package that supports PhpSpec v3 will be used.